packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -33,6 +33,11 @@ const propTypes = {
    */
   id: PropTypes.string.isRequired,
 
+  /**
+   * Boolean specifying whether or not the component should have zebra striping for rows.
+   */
+  isStriped: PropTypes.bool,
```

Comment:
This should not be a prop for the WorklistDataGrid.  The WorklistDataGrid should always be striped.